<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QDialog" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QForm
{
	background-color: black;
}</string>
  </property>
  <widget class="QFrame" name="LeftPanel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>80</y>
     <width>181</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: lightgray;
    border: 1px solid #31363F;
    border-radius: 20px;
}

QFrame:hover {
    background-color: darkgray;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="SaveContactButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #31363F;
    color: white;
    border: none;
    padding: 8px 16px;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #CED4DA;
    color: #4CAF50;
}</string>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelAddContact">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #31363F;
    color: white;
    border: none;
    padding: 8px 16px;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #CED4DA;
    color: #4CAF50;
}</string>
    </property>
    <property name="text">
     <string>CANCLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #31363F;
    color: white;
    border: none;
    padding: 8px 16px;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #CED4DA;
    color: #4CAF50;
}</string>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="RightPanel">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>80</y>
     <width>481</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background-color: #272829;
	border: 1px solid #31363F;
	border-radius:20px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="OperationHeading">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
	background-color: lightgray;
	color: #31363F;
}</string>
    </property>
    <property name="text">
     <string>UPDATE INFORMATION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="FirstNameHeading">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="ContactNumberHeading">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Contact no</string>
    </property>
   </widget>
   <widget class="QLabel" name="HouseHeading">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>House no.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstNameInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 10px;
    color: darkgray; 
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastNameInput">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 10px;
    color: darkgray; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PhoneNumberInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 10px;
    color: darkgray; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailAddressInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 10px;
    color: darkgray; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HouseInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 10px;
    color: darkgray; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CountryInput">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 10px;
    color: darkgray; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CityInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 10px;
    color: darkgray; 
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="StreetInput">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 10px;
    color: darkgray; 
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="LastNameHeading">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="EmailHeading">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="StreetHeading">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Street no.</string>
    </property>
   </widget>
   <widget class="QLabel" name="CityHeading">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QLabel" name="CountryHeading">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: white;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Country</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="UpperPanel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: lightgray;
    border: 1px solid #31363F;
    border-radius: 20px;
}

QFrame:hover {
    background-color: darkgray;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Heading">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: darkgray;
	border: none;
	color: #31363F;
}</string>
    </property>
    <property name="text">
     <string>CONTACT BOOK APP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
